Library/Homebrew/cmd/info.rb

```diff
@@ -126,6 +130,11 @@ def print_info
           if (reason = MissingFormula.reason(f))
```

Comment:
You mean create a "reason" method on `MissingFormula` that searches for a cask and returns it's info printout? I think it would be a little cleaner. 

But there's a similar call to `MissingFormula.reason()` in `Homebrew/cmd/install.rb`, this could cause an attempting to call `brew install` on a missing formula to search for a cask with the same name and print it's info. Is this desirable?

Reply:
@zachauten I think the reason method could suggest a Cask when a formula isn't available. In this case if you want it to be handled automatically for `brew info` to show casks I think it's still worth adding to `MissingFormula` but it would be good to search for a cask before you search for `MissingFormula` (which will take longer and print output about e.g. searching which seems unnecessary). Additionally, the output here for searching for casks is only necessary if it takes >1s on your machine.